Vizzuality/trase
# Performance/replace flow nodes mv with table

## Pivotal Tracker

https://www.pivotaltracker.com/story/show/169828789
https://www.pivotaltracker.com/story/show/169828841
https://www.pivotaltracker.com/story/show/169828825

## Description

In order to speed up the refresh process, as well as to enable fixing some weird bugs in dashboards filtering resulting from lack of support for year filtering, the dashboards mviews were changed into tables.

## Testing instructions

Run migrations - it's one of the long ones, but I think it's now faster than it used to be to refresh the dashboards tables. Then check if dashboards filtering works as before.
